index.test.js

```diff
@@ -1,5 +1,7 @@
 const { test } = require('tap');
```

Comment:
@hawyar - I haven't used tap before. Looks cool. Tell me about why you like it!

Reply:
Small API and easy to use. I don't like connecting three tools for some non-complex testing. Tap gets the job done without a headache.